ui-show/src/main/java/com/michaldrabik/ui_show/cases/ShowDetailsTranslationCase.kt

```diff
@@ -17,10 +17,10 @@ class ShowDetailsTranslationCase @Inject constructor(
 
   suspend fun loadTranslation(show: Show): Translation? =
     withContext(dispatchers.IO) {
-      val language = translationsRepository.getLanguage()
-      if (language == DEFAULT_LANGUAGE) {
+      val locale = translationsRepository.getLocale()
+      if (locale == AppLocale.default()) {
```

Comment:
These checks (and also the ones with != ) will need to be updated because now it will only check against "en-us". And for users from ex. United Kingdom (en_GB) it would always try to load english translation which does not make sense really.

Was also thinking of getting rid of these checks accross the code and put them into TranslationRepository itself. If english language is requested return null or empty list. 🤔